core/src/main/java/tc/oc/pgm/filters/dynamic/FilterMatchModule.java

```diff
@@ -66,13 +71,24 @@
 public class FilterMatchModule implements MatchModule, FilterDispatcher, Tickable, Listener {
 
   private final Match match;
+  private final ContextStore<?> filterContext;
   private final List<Class<? extends Event>> listeningFor = new LinkedList<>();
-  private final PriorityQueue<TimeFilter> timeFilterQueue = new PriorityQueue<>();
+
+  private final Map<ReactorFactory<? extends ReactorFactory.Reactor>, ReactorFactory.Reactor>
```

Comment:
```suggestion
  private final Map<ReactorFactory<?>, ReactorFactory.Reactor>
```
Reactor factory's generic type is already bounded, no need to set a bound here